src/connector/src/sink/mod.rs

```diff
@@ -303,6 +303,7 @@ pub struct SinkMetrics {
     pub iceberg_rolling_unflushed_data_file: LabelGuardedIntGauge<3>,
     pub iceberg_position_delete_cache_num: LabelGuardedIntGauge<3>,
     pub iceberg_partition_num: LabelGuardedIntGauge<3>,
+    pub iceberg_write_bytes: LabelGuardedIntCounter<3>,
```

Comment:
Oops, I remember https://github.com/risingwavelabs/risingwave/pull/18758 has improved the sink metrics, so the nimtable branch is out of date. I think we'd better implement this feature in the main branch first to avoid potential conflicts.